### ISSUE TITLE
Document update proposal 

### ISSUE THREAD

**Issue by Masutani** [leonawicz/mapmate#4]
Thank you for your perfect solution for map animation !
I found some tiny glitches  in mapmate documents 👍 

1) library
https://leonawicz.github.io/mapmate/
apparently 
devtools::install_github("leonawicz/rvtable")
should be
devtools::install_github("leonawicz/mapmate")

2) saveGIF only accepts expression
at https://leonawicz.github.io/mapmate/articles/networks.htm
saveGIF(for (i in seq_along(gglist)) print(gglist[[i]]), "network3D.gif", interval = 1/20, 
    ani.width = 600, ani.height = 600)
should have curly bracket :
saveGIF( { for (i in seq_along(gglist)) print(gglist[[i]]) } , "network3D.gif", interval = 1/20, 
    ani.width = 600, ani.height = 600)

3) ImageMagick
I have checked with latest version of ImageMagick.
It has some problem on very common Windows file names trap.
ani.options(convert = "C:/Program Files/ImageMagick-7.0.7-Q16/convert.exe")
should be :
ani.options(convert = "C:/Progra~1/ImageMagick-7.0.7-Q16/convert.exe")
The upper one yield error like : " 'C:/Program' cannot be recognized as executable "

Thank you.

**Comment by leonawicz** [comment]
Thank you for your feedback.

I have fixed the package name copy/paste errors in the README file.

I could not duplicate the issue with `saveGIF` requiring a strict expression, even by breaking the loop above into multiple lines. However, what you say is reasonable and probably generally true, so I wrapped the `animation` examples in `{}` anyway now.

I also fixed the `Program Files` path issue. I thought it would be okay with ImageMagick in my system path, but apparently not.

I've done a small version increment to v0.3.1. It is online now.

Thank you!
Matt
